### PR TITLE
Adding RNs for mmc 3.8.3

### PR DESCRIPTION
**Pull request by sleslie23**
@mcocaro

### CODE REVIEW

File: release-notes/v/latest/mule-management-console-3.8.3.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+[INFO]
+Version 3.8.3 of MMC is compatible with Mule runtime V3.5 and above.
+
+
+== Resolved
+
+* SE-5408 - Wrong application name displayed for business events
+* SE-4762 - MMC doesn't escape correctly flow names
+* SE-4582 - MMC Deployments Tab Pagination
```
Comment (mcocaro) [inline release-notes/v/latest/mule-management-console-3.8.3.adoc]: Change to: MMC pagination for any listing does not work properly when filtering.

File: release-notes/v/latest/mule-management-console-3.8.3.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+* SE-4133 - log4j2 appender for scripts is not present in log configuration
+* SE-3312 - Unregistered servers show ${project.version} as Agent version
+* SE-1823 - Alert notification for Thread Pool type filter is not sending email messages
+* SE-181 - No email notifications generated by MMC
+* SE-1659 - Survivors space max size is incorrectly reported
+* SE-1582 - MMC fails to send to email if MS Exchange server requires authentication on SMTP
+* SE-846 - MMC Displays MaxMemory and Total Available Memory without decimals
+* SE-533 - Alert notifications with the filter 'Type' not working
+
```
Comment (mcocaro) [inline release-notes/v/latest/mule-management-console-3.8.3.adoc]: Add the following:

SE-6195 - Session Id stripped from clear text logs

File: release-notes/v/latest/mule-management-console-3.8.3.adoc [hunk 3]
```diff
@@ -0,0 +1,27 @@
+= Mule Management Console 3.8.3
+:keywords: release notes, mmc mule management console
+
+The Mule Management Console, also known as MMC, is a platform for managing Mule applications deployed on premises. Currently, no new functionality is being added to the MMC, this release includes only bug fixes and security fixes. 
+
+[INFO]
+Version 3.8.3 of MMC is compatible with Mule runtime V3.5 and above.
```
Comment (mcocaro) [inline release-notes/v/latest/mule-management-console-3.8.3.adoc]: I would word it as: is compatible with Mule Runtime V3.X starting from V3.5 and above.

Comment (sleslie23) [reply release-notes/v/latest/mule-management-console-3.8.3.adoc]: Fixed.

File: release-notes/v/latest/mule-management-console-3.8.3.adoc (hunk 1 above)
Comment (sleslie23) [reply release-notes/v/latest/mule-management-console-3.8.3.adoc]: Fixed.

File: release-notes/v/latest/mule-management-console-3.8.3.adoc (hunk 2 above)
Comment (sleslie23) [reply release-notes/v/latest/mule-management-console-3.8.3.adoc]: Added.

**Pull request merged**